api-flask/app/main.py

```diff
@@ -1,290 +1,340 @@
-
-    return Response(json.dumps(alert, cls=AlchemyEncoder), mimetype='application/json')
-
-
-@app.route('/alerts', methods=['POST'])
-def write_alerts():
-    """Post new alerts."""
-    if not request.is_json:
-        raise InternalServerError('InvalidInput')
-
-    try:
-        data = request.json
-        alert = AlertModel(**data)
-        alert_db.write(alert)
-
-    except rasterio.errors.RasterioError as e:
-        logger.error(e)
-        raise e
-
-    return Response(response='Success', status=200)
-
-
-@app.route('/demo', methods=['GET'])
-@timed
-def stats_demo():
-    """Return examples of zonal statistics."""
-    # The GET endpoint is used for demo purposes only
-    geotiff_url = urlunparse(
-        ParseResult(
-            scheme='https',
-            netloc='mongolia.sibelius-datacube.org:5000',
-            path='/',
-            params='',
-            query=urlencode({
-                'service': 'WCS',
-                'request': 'GetCoverage',
-                'version': '1.0.0',
-                'coverage': 'ModisAnomaly',
-                'crs': 'EPSG:4326',
-                'bbox': '86.5,36.7,119.7,55.3',
-                'width': '1196',
-                'height': '672',
-                'format': 'GeoTIFF',
-                'time': '2020-03-01'
-            }),
-            fragment='',
-        )
-    )
-
-    zones_url = urlunparse(
-        ParseResult(
-            scheme='https',
-            netloc='prism-admin-boundaries.s3.us-east-2.amazonaws.com',
-            path='mng_admin_boundaries.json',
-            params='',
-            query='',
-            fragment=''
-        )
-    )
-
-    geotiff = cache_file(
-        prefix='raster_test',
-        url=geotiff_url,
-        extension='tif'
-    )
-
-    zones = cache_file(
-        prefix='zones_test',
-        url=zones_url
-    )
-
-    geojson_out = request.args.get('geojson_out', 'False')
-    group_by = request.args.get('group_by', None)
-
-    geojson_out = strtobool(geojson_out)
-
-    intersect_comparison_string = request.args.get('intersect_comparison', None)
-
-    intersect_comparison = None
-    if intersect_comparison_string is not None:
-        intersect_comparison = validate_intersect_parameter(
-            intersect_comparison_string
-        )
-
-    features = _calculate_stats(
-        zones,
-        geotiff,
-        stats=['min', 'max', 'mean', 'median', 'sum', 'std'],
-        prefix='stats_',
-        group_by=group_by,
-        geojson_out=geojson_out,
-        wfs_response=None,
-        intersect_comparison=intersect_comparison
-    )
-
-    # TODO - Properly encode before returning. Mongolian characters are returned as hex.
-    return jsonify(features)
-
-
-if __name__ == '__main__' and getenv('FLASK_ENV') == 'development':
-    PORT = int(getenv('PORT', 80))
-    # Only for debugging while developing
-    app.run(host='0.0.0.0', debug=True, port=PORT)
+"""Flask API for geospatial utils."""

```

Comment:
It is very odd that the whole file is seen as new. Can you try to fix this git problem?